src/Grpc.Net.Client/Balancer/ChannelState.cs

```diff
@@ -0,0 +1,63 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+#if SUPPORT_LOAD_BALANCING
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using Grpc.Core;
+using Grpc.Net.Client.Configuration;
+
+namespace Grpc.Net.Client.Balancer
+{
+    /// <summary>
+    /// Represents the state for a channel. This is created from results returned by a <see cref="Resolver"/>.
+    /// </summary>
+    public sealed class ChannelState
```

Comment:
`ChannelState` is the name other implementations use for this type. It is good to be consistent but it can change if required.

I think it is ok because:
* Most apps won't be using the Grpc.Net.Client.Balancer namespace. It is only used when implementing a new resolver or load balancer
* ChannelState is in Grpc.Core. Most apps won't reference Grpc.Net.Client and Grpc.Core, so won't see a conflict.

Reply:
Fair enough.